packages/server/src/controllers/accounts.ts

```diff
@@ -3,36 +3,57 @@ import { insertOneAccount, retrieveAccountById } from "../db/accounts";
 import { hashPassword, verifySession } from "../utils/auth";
 import { extractSessionId } from "../middlewares/authenticate";
 import { retrieveAccountBySessionId } from "../db/sessions";
+import { logger, sanitizeError } from "../logger";
 
 export const createAccount = async (ctx: Context) => {
   const { firstName, lastName, email, password } = ctx.request.body;
   const hashedPassword = await hashPassword(password);
   try {
-    await insertOneAccount(firstName, lastName, email, hashedPassword);
+    const [accountId] = await insertOneAccount(
+      firstName,
+      lastName,
+      email,
+      hashedPassword,
+    );
+
+    const successMessage = "New account created";
+    logger.info(successMessage, {
```

Comment:
I suggest to remove unnecessary variable declaration as you use it only one time
```suggestion
    logger.info("New account created", {
```